src/libraries/System.Private.CoreLib/src/System/TimeZoneInfo.Unix.Android.cs

```diff
@@ -265,9 +265,13 @@ public AndroidTzData()
             //
             // Once the timezone cache is populated with the IDs, we reference tzlookup id tags
             // to determine if an id is backwards and label it as such if they are.
-            private static void FilterBackwardIDs(string tzFileDir, out HashSet<string> tzLookupIDs)
+            private static bool FilterBackwardIDs(string tzFileDir, out HashSet<string> tzLookupIDs)
             {
                 tzLookupIDs = new HashSet<string>();
```

Comment:
maybe delay allocation after the file exist check?

Reply:
Done, thanks for the suggestion!